packages/devtools-reps/src/reps/stubs/window.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const structs = new Map();
```

Comment:
Could it be named `stubs` instead of `structs` so we're consistent with what we have in the other files ?